dash/development/base_component.py

```diff
@@ -223,7 +226,7 @@ def __init__(self, {default_argtext}):
 
         def __repr__(self):
             if(any(getattr(self, c, None) is not None for c in self._prop_names
-                   if c is not self._prop_names[0])):
+                   if (c is not self._prop_names[0] and (not c[:5] in ['data_', 'aria_'])))):
```

Comment:
`__repr__` should print out a string that can be copied and pasted back into a python program to get the same result. In this case, it looks like we have:
```
>>> html.Div('test', data_foo='bar')
Div(children='test', data-foo='bar')
```